app/assets/javascripts/govuk_publishing_components/components/checkboxes.js

```diff
@@ -40,5 +40,5 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         $('#' + parent_id).prop("checked", false);
       }
     };
-  }
+  };
```

Comment:
Could consider running StandardJS on these files: https://gds-way.cloudapps.digital/build-services.html#source-formatting-and-linting